appveyor.yml

```
@@ -38,7 +37,21 @@ deploy:
 
   - provider: GitHub
     tag: $(APPVEYOR_REPO_TAG_NAME)
+    description: ''
     auth_token:
         secure: Otbl8p8qCwciDqJgSWCyN0Arfs5XS1CwiHcK+r0F6uz9Rxt4gzBFvlc3cjPV3NxR
     on:
-        APPVEYOR_REPO_TAG: true
\ No newline at end of file
+        APPVEYOR_REPO_TAG: true
+
+after_deploy:
+  - ps: |
+      # Ensure we only push on tagged builds
+      if ($env:APPVEYOR_REPO_TAG -eq "true") {
+        foreach ($artifactName in $artifacts.keys) {
+          $path = $artifacts[$artifactName].path
+          # Only push the NestingContently Umbraco package
+          if ($path -match "Nesting_Contently_[0-9.]+\.zip$") {
+            umbpack push $path -k $env:UmbPackKey -a current
+          }
+        }
```

If you also want to push the ValueConverters package, the first push can archive all existing packages and the second do a push of this additional package without setting it as new current version:

```suggestion
        foreach ($artifactName in $artifacts.keys) {
          $path = $artifacts[$artifactName].path
          # Push the NestingContently Umbraco package and archive all existing packages
          if ($path -match "Nesting_Contently_[0-9.]+\.zip$") {
            umbpack push $path -k $env:UmbPackKey -a *
          }
        }
        foreach ($artifactName in $artifacts.keys) {
          $path = $artifacts[$artifactName].path
          # Push the NestingContently ValueConverters Umbraco package without setting it as the new current version
          if ($path -match "Nesting_Contently_Value_Converters_[0-9.]+\.zip$") {
            umbpack push $path -c false -k $env:UmbPackKey
          }
        }
```

---

If you want the ValueConverters Umbraco package to be installed via the back-office (instead of downloading the ZIP-file and manually installing), it probably needs its own Our Umbraco package page and a different API key.